src/main/java/build/buildfarm/common/redis/RedisHashtags.java

```diff
@@ -31,8 +35,10 @@ public class RedisHashtags {
    * @return A valid queue name for one of the internal queues.
    * @note Suggested return identifier: queueName.
    */
-  public static String hashedName(String name, String hashtag) {
-    return "{" + hashtag + "}" + name;
+  public static String hashedName(@Nonnull String name, @Nullable String hashtag) {
```

Comment:
we assume that non-@Nullable fields are not accepted to be null - definitely have some outliers, but I don't want to start going down the route of tagging *everything* one way or the other.

Reply:
No worries, I can revert all of the `javax.annotations.*` in this class.